src/current/_includes/releases/cloud/2023-11-29.md

```diff
@@ -0,0 +1,6 @@
+## November 29, 2023
```

Comment:
I can't see this on the cloud release note page... am I looking in the wrong place? https://deploy-preview-18047--cockroachdb-docs.netlify.app/docs/releases/cloud.html

There is a SCIM note on that page ^ but it's under Nov 16 and there is a typo on "Gemerally Available"

Reply:
There are also a couple of floating hyphen bullet point markers in the preceding dates under "General changes" — this may not be to do with your PR though.

Reply:
> I can't see this on the cloud release note page... am I looking in the wrong place? https://deploy-preview-18047--cockroachdb-docs.netlify.app/docs/releases/cloud.html
> 
> There is a SCIM note on that page ^ but it's under Nov 16 and there is a typo on "Gemerally Available"

I didn't notice the build error that stopped Netlify from updating with this change -- the preview is now stale. I'm investigating the build error.